.pipelines/prod-release.yml

```diff
@@ -3,16 +3,69 @@ pr: none
 trigger: none
 
 stages:
-- stage: Deploy_USSector
+- stage: Deploy_LowTrafficSector
   dependsOn: []
+  displayName: 🚀 Deploy Low Traffic Sector
+  jobs:
+  - template: ./templates/template-job-deploy-azure-env.yml
+    parameters:
+      environment: RP-Prod-LowTrafficSector
+      locations:
+      - australiaeast
+      configFileName: prod-config.yaml
+      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+      vsoProjectID: $(vso-project-id)
+      vsoConfigPipelineID: $(vso-config-pipeline-id)
+      vsoDeployerPipelineID: $(vso-deployer-pipeline-id)
+      azureDevOpsE2EJSONSPN: $(aro-v4-e2e-devops-spn)
+      e2eSubscription: $(e2e-subscription)
+- stage: Deploy_USSector
+  dependsOn: [Deploy_LowTrafficSector]
   displayName: 🚀 Deploy USSector
   jobs:
-  - template: ./templates/template-job-deploy-azure-region.yml
+  - template: ./templates/template-job-deploy-azure-env.yml
     parameters:
-      environment: RP-Prod
+      environment: RP-Prod-USSector
       locations:
       - eastus
       - eastus2
+      - southcentralus
+      - westus2
```

Comment:
will need to find a way to disable individual e2es where quota is not available

Reply:
seems like westus2 is the only one in this case

Reply:
As opposed to skipping the e2es I'm switching to allowing running the next stage even if the previous one failed

Reply:
perfect